config/environments/test.rb

```diff
@@ -39,4 +39,20 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  # Setup mailer
+
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
```

Comment:
> I think you should config mailer in a separate PR.

I agree. By the way, I don't think we have anything to manage our secrets right now yea?